app/main/services/query_builder.py

```diff
@@ -148,12 +159,13 @@ def or_field_filters(field_name, field_values):
     (https://www.elastic.co/guide/en/elasticsearch/reference/1.6/query-dsl-terms-filter.html)
 
     """
-    terms = [strip_and_lowercase(value) for value in field_values]
-    return [{
-        "terms": {
-            field_name: terms
-        }
-    }]
+    return [
```

Comment:
We're no longer stripping/lower casing the field values. I'm assuming this is because the values are defined by us in the frameworks repo and aren't user input?

Reply:
Nope, we're just letting ES do the normalization for us (`"normalizer": "filter_normalizer"`)

Reply:
I suppose the only thing here is that we therefore have to repeat it in every mapping. Which isn't okay if _we_ rely on it in some way. It's probably fine if only something external relies on it, though, which is I think the case?

Reply:
Point taken, but there is an assumption externally that we've done this, because the frontends also strip the non-alpha characters when passing their arguments into URLs etc.

Reply:
Yup, but then again there are numerous things that frontends "assume" about the particular mapping that they're working against.